robotouille/env.py

```diff
@@ -327,18 +57,46 @@ def build_input_json(domain_json):
 
     return input_json
 
+def build_gamemode(environment_json, domain_json, state, layout, animate):
+    """
+    This function builds the gamemode of the environment. 
+
+    Args:
+        environment_json (dict): The environment dictionary
+        domain_json (dict): The domain dictionary
+        state (State): The state of the environment
+        layout (list): A 2D list of station names representing where stations are in the environment.
+        animate (bool): Whether or not to animate the movement of the players.
+
+    Returns:
+        gamemode(GameMode): The gamemmode of the environment
```

Comment:
space between `gamemode (GameMode)` and misspelled gamemode.

Reply:
added space and fixed typo